pkgs/development/tools/pgloader/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchurl, makeWrapper, sbcl, sqlite, freetds, libzip, curl, git, cacert, openssl }:
+stdenv.mkDerivation rec {
+  pname = "pgloader";
+  version = "3.6.1";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/dimitri/pgloader/releases/download/v3.6.1/pgloader-bundle-3.6.1.tgz";
+    sha256 = "1sm8xmq30d1biin5br0y3vrv4fydbrzfqglz1hnvrkdyxrg7d6f9";
+  };
+
+  buildInputs = [ sbcl cacert sqlite freetds libzip curl git openssl makeWrapper ];
```

Comment:
Aren't `git` and `makeWrapper` native build inputs?

Reply:
Good point. Force pushed.